dotcom-rendering/README.md

```diff
@@ -73,7 +73,7 @@ If you're new to JavaScript projects, if you're trying to integrate with other a
 | <img alt="Emotion CSS-in-JS" src="./docs/images/logo-emotion.png" width="350" />                                           | Emotion is css-in-js library, DCR uses the `css` tagged template literal style to allow CSS copy-pasting.                                                                                                                                                                                                                                         |
 | <img alt="Typescript" src="./docs/images/logo-typescript.png" width="350" />                                               | DCR is written in Typescript. You can see the [block element types](./src/lib/content.d.ts) as an example of our Typescript types.                                                                                                                                                                                                                |
 | <img alt="Express" src="./docs/images/logo-express.png" width="350" />                                                     | We use Express as a very thin server to communicate with the Frontend endpoint.                                                                                                                                                                                                                                                                   |
-| <img alt="Storybook" src="./docs/images/logo-storybook.jpg" width="350" />                                                 | We use [storybook to generate component variations and 'layouts'](https://5dfcbf3012392c0020e7140b-borimwnbdl.chromatic.com/?path=/story/*) that are then visual regression tested in Chromatic. You'll notice `.stories.` files in the repo for components that define the variations of components as defined by the component props.           |
+| <img alt="Storybook" src="./docs/images/logo-storybook.jpg" width="350" />                                                 | We use [storybook to generate component variations and 'layouts'](https://main--63e251470cfbe61776b0ef19.chromatic.com/?path=/story/*) that are then visual regression tested in Chromatic. You'll notice `.stories.` files in the repo for components that define the variations of components as defined by the component props.                |
```

Comment:
Actually this link does not work: https://main--63e251470cfbe61776b0ef19.chromatic.com/?path=/story/*